Nodejs/Product/Nodejs/ProjectTemplates/TypeScriptConsoleApp/package.json

```diff
@@ -6,7 +6,12 @@
   "author": {
     "name": ""
   },
+  "scripts": {
+    "build": "tsc --build",
+    "clean": "tsc --build --clean"
```

Comment:
Naïve question: why no `start` for console apps?

Reply:
`start` serves no purpose for any project as far as I can see. My best guess is that it might have been added for express apps long ago as part of the "express" template.